### PR TITLE
[OMA-948] - Integrate mopub adtag demo to CircleCI firebase publication

### PR DESCRIPTION
**Pull request by PabloGarciaAldehuela**
- Updated material dependency in build.gradle
- Added config.yml file to proyect

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+      - *ruby_yum
+      - *bundler_update
+      - *ruby_dependencies
+      - *save_gradle_cache
+      - *save_gems_cache
+      - run:
+          name: Deploy to Firebase App Distribution
+          command: bundle exec fastlane deploy_to_firebase
+
+  publish_bintray:
```
Comment (eros902002) [inline .circleci/config.yml]: You don't need this task for the mopub demo

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+      - *ruby_yum
+      - *bundler_update
+      - *ruby_dependencies
+      - *save_gradle_cache
+      - *save_gems_cache
+      - run:
+          name: Run Lint checks
+          command: bundle exec fastlane lint_check
+
+  send_coverage_report:
```
Comment (eros902002) [inline .circleci/config.yml]: This step is not needed either

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+          name: Run unit tests
+          command: bundle exec fastlane unit_tests
+      - store_artifacts:
+          path: app/build/reports/
+          destination: /reports/
+      - store_test_results:
+          path: app/build/test-results/
+          destination: /test-results/
+
+  lint_check:
```
Comment (eros902002) [inline .circleci/config.yml]: Not needed for this demo

File: .circleci/config.yml [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+      name: Update bundler
+      command: gem install bundler --version 1.16.3
+
+  ruby_dependencies: &ruby_dependencies
+    run:
+      name: Download Ruby Dependencies
+      command: bundle check || bundle install --path vendor/bundle
+
+jobs:
+  test_unit:
```
Comment (eros902002) [inline .circleci/config.yml]: Not needed for this demo

File: .circleci/config.yml [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+      - *save_gradle_cache
+      - *save_gems_cache
+      - run:
+          name: Publish to Bintray
+          command: ./gradlew bintrayUpload
+
+workflows:
+  version: 2
+  build_test_deploy:
+    jobs:
```
Comment (eros902002) [inline .circleci/config.yml:82]: From here we should remove all jobs except the deploy_firebase one

File: .circleci/config.yml [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+          requires:
+            - test_unit
+
+      - deploy_firebase:
+          filters:
+            branches:
+              only:
+                - master
+          requires:
+            - test_unit
```
Comment (eros902002) [inline .circleci/config.yml]: We should remove the requirement since the test unit job will be removed

Comment (PabloGarciaAldehuela) [reply .circleci/config.yml]: I did all the changes mentioned above

**Review by eros902002 — APPROVED** [review]
Looks good

**Pull request merged**